content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -32,32 +32,60 @@ Using the latest version of kubectl helps avoid unforeseen issues.
 
 1. Download the latest release with the command:
 
+    ```bash
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
nit: Is this code block lined up with the letter "D"?

Reply:
> nit: Is this code block lined up with the letter "D"?

It is now :) 

> nit: The H2 heading ,"Install kubectl on Linux" could have text following the heading. It does provide a bucket for all "Linux" install instructions, otherwise the heading does not contribute too much.

I agree with you here, but am hesitant to write up any copy just yet because I think I'd like to open a new issue regarding using tabs on this page (as per https://github.com/kubernetes/website/pull/25259#issuecomment-735020297 & https://github.com/kubernetes/website/pull/25259#pullrequestreview-554120404). I think using tabs would probably negate the need to have copy here, or it would be enough of a restructure that the copy would likely need to change.

Reply:
Here's that issue: https://github.com/kubernetes/website/issues/26096 :)